src/app/zones/views/ZoneDetailView.vue

```diff
@@ -102,14 +106,79 @@
             v-if="props.data.zoneInsight.subscriptions.length > 0"
           >
             <h2>{{ t('zone-cps.detail.subscriptions') }}</h2>
-
-            <KCard class="mt-4">
-              <SubscriptionList
-                :subscriptions="props.data.zoneInsight.subscriptions"
+            <AppCollection
+              :headers="[
+                { ...me.get('headers.name'), label: 'Name', key: 'name' },
+                { ...me.get('headers.connected'), label: 'Connected', key: 'connected' },
+                { ...me.get('headers.disconnected'), label: 'Disconnected', key: 'disconnected' },
+                { ...me.get('headers.responses'), label: `Responses (sent/ack'ed)`, key: 'responses' },
```

Comment:
I think I'd add the version as well. It's useful when you want to follow when you are upgrading

Reply:
Adding the below extra column

![Screenshot 2024-07-30 at 16 36 17](https://github.com/user-attachments/assets/7e84d65e-0c72-4d65-acd6-735fcef014e3)

Reply:
This is done like the above screengrab, resolving